source/includes/_authentication.md

```diff
@@ -2,10 +2,15 @@
 
 The Monzo API implements [OAuth 2.0](http://oauth.net/2/) to allow users to log in to applications without exposing their credentials. The process involves several steps:
 
-1. [**Acquire**](#acquire-an-access-token) an access token, and optionally a refresh token
+1. [**Acquire**](#acquire-an-access-token) an access token, and optionally a refresh token.
 2. [**Use**](#authenticating-requests) the access token to make authenticated requests
 3. If you were issued a refresh token: [**refresh**](#refreshing-access) the access token when it expires
 
+<aside class="warning">
+<strong>Strong Customer Authentication</strong><br/>
+We don't grant any permissions to the access token util the owner of the account your client wants to access has approved access to their data in the Monzo app. Your user will receive a push notification after autheticating with their email.
```

Comment:
typos `util` > `until`, `autheticating` > `authenticating`
also might want to add the word `that` > `the account that your client wants to access`